build/index.js

```diff
@@ -228,13 +228,29 @@ async function buildDocumentFromURL(url) {
 async function buildLiveSamplePageFromURL(url) {
   const [documentURL, sampleID] = url.split("/_samples_/");
   const document = Document.findByURL(documentURL);
-  const liveSamplePage = kumascript.buildLiveSamplePage(
-    document.url,
-    document.metadata.title,
-    (await kumascript.render(document.url))[0],
-    { id: sampleID, createFlaw() {} }
-  );
-  return liveSamplePage.html;
+  if (!document) {
+    return `unable to find ${documentURL}`;
```

Comment:
I think we should turn this into a 
```
throw new Error(`No document found for ${documentURL}`);
```
At which point we should also add some error handling / status sending into the server code.

Reply:
@Gregoor Yep, that's better! I'll do that.